qna/General-entitlement-primary-contact.md

```diff
@@ -0,0 +1,14 @@
+## Question
+
+Was ist der Primärkontakt oder Primary Contact und wie kann ich diesen ändern?
+
+## Metadata tags
+
+lang-de, market-de, shop, PosCreator, PosDealer, Consultant
+
+## Answer
+
+Ihr Primärkontakt wird mit der Registrierung als Benutzer im fiskaltrust.portal erstellt. Mit dem Primärkontakt besitzen Sie alle Rechte. Legen Sie unter <kbd>Firmenname</kbd>  &rarr; <kbd>Mitarbeiter</kbd> weitere Benutzer an, können Sie den Datensatz anklicken und gezielt Rollenrechte vergeben.
+Dort finden Sie auch die Schaltfläche <kbd>Primärer Kontakt</kbd>, mit welcher Sie einen Benutzer als neuen Primärkontakt festlegen.
+Mit dem Wechsel des Primärkontakts werden für den gewählten Benutzer alle Rechte frei- und Ihre Rollenrechte komplett abgeschaltet. Damit ist **keine Anmeldung im Portal** mehr möglich. Der neue Primärkontakt kann Ihrem Benutzer wieder Rollenrechte zuweisen, falls gewünscht.
```

Comment:
Mit dem Wechsel des Primärkontakts werden für den vorherigen Primärkontakt alle Rechte widerrufen. Damit ist **keine Anmeldung im Portal** mehr möglich. Um zumindest die Anmeldung zu ermöglichen muss das Recht `Lesen` vergeben werden. Der neue Primärkontakt kann in der Folge dem vorherigen Primärkontakt wieder Rollenrechte zuweisen.